src/main/java/bio/terra/stairway/Database.java

```diff
@@ -172,8 +150,8 @@ public void complete(FlightContext flightContext) {
             connection.setAutoCommit(false);
 
             statement.setString("output_parameters", flightContext.getWorkingMap().toJson());
-            statement.setBoolean("succeeded", flightContext.getResult().isSuccess());
-            statement.setString("error_message", flightContext.getResult().getErrorMessage());
+            statement.setString("status", flightContext.getFlightStatus().name());
+            statement.setString("error_message", flightContext.getResult().getErrorMessage().orElse(null));
```

Comment:
I want to set the error_message column to null, to indicate that there is no error message. The way to do that is to pass null into the setXxx call.

The way I had it before, errorMessage.isPresent() would say "true", even if there was no error message, because I was storing an empty string into the database.